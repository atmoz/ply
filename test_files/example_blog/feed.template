{{ $RFC3339 := "2006-01-02T15:04:05Z07:00" -}}
<?xml version="1.0" encoding="UTF-8"?>
<feed xmlns="http://www.w3.org/2005/Atom">
  <title>Example blog</title>
  <link href="localhost"></link>
  <id>example</id>
  <updated>{{ $RFC3339 }}</updated>
  <summary>example</summary>
  <author>
    <name>example</name>
  </author>

  {{ range .Sitemap -}}
  {{ if (not (eq .Path "index.html")) }}
  <entry>
    <title>{{ .Title }}</title>
    <link href="{{ .Path }}"></link>
    <id>{{ .Path }}</id>
    <updated>{{ timeFormat (timeParse $RFC3339 .Meta.date) $RFC3339 }}</updated>
    <summary type="html"><![CDATA[{{ regexFind `(?Us:<p>.*</p>)` .Content }}]]></summary>
  </entry>
  {{- end }}
  {{- end }}
</feed>

