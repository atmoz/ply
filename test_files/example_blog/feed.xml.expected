<?xml version="1.0" encoding="UTF-8"?>
<feed xmlns="http://www.w3.org/2005/Atom">
  <title>Example blog</title>
  <link href="localhost"></link>
  <id>example</id>
  <updated>2006-01-02T15:04:05Z07:00</updated>
  <summary>example</summary>
  <author>
    <name>example</name>
  </author>

  
  <entry>
    <title>My first post</title>
    <link href="my-first-post.html"></link>
    <id>my-first-post.html</id>
    <updated>2018-08-17T16:32:00Z</updated>
    <summary type="html"><![CDATA[<p>Content goes here</p>]]></summary>
  </entry>
  <entry>
    <title>My last post</title>
    <link href="my-last-post.html"></link>
    <id>my-last-post.html</id>
    <updated>2018-08-17T16:35:00Z</updated>
    <summary type="html"><![CDATA[<p>Content goes here</p>]]></summary>
  </entry>
</feed>

